commit 06ffb1175eb87684eaccb4fd7ac9d4936ca9b8f7
Author: David Carlier <devnexen@gmail.com>
Date:   Wed Mar 27 16:08:42 2019 +0000

    BUILD/MINOR: listener: Silent a few signedness warnings.
    
    Silenting couple of warnings related to signedness, due to a mismatch of
    signed and unsigned ints with l->nbconn, actconn and p->feconn.
    
    (cherry picked from commit 5671662f08536c979ff91bc46f94d086bf286540)
    [wt: s/_HA/HA/]
    Signed-off-by: Willy Tarreau <w@1wt.eu>
    (cherry picked from commit f5e9bf696b0b46c140d742cee23d0d54df66bb2f)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/src/listener.c b/src/listener.c
index 68c84fbe..821c931a 100644
--- a/src/listener.c
+++ b/src/listener.c
@@ -499,7 +499,7 @@ void listener_accept(int fd)
 				goto end;
 			}
 			next_conn = count + 1;
-		} while (!HA_ATOMIC_CAS(&l->nbconn, &count, next_conn));
+		} while (!HA_ATOMIC_CAS(&l->nbconn, (int *)&count, next_conn));
 
 		if (next_conn == l->maxconn) {
 			/* we filled it, mark it full */
@@ -536,7 +536,7 @@ void listener_accept(int fd)
 					goto end;
 				}
 				next_actconn = count + 1;
-			} while (!HA_ATOMIC_CAS(&actconn, &count, next_actconn));
+			} while (!HA_ATOMIC_CAS(&actconn, (int *)&count, next_actconn));
 
 			if (unlikely(next_actconn == global.maxconn)) {
 				limit_listener(l, &global_listener_queue);
